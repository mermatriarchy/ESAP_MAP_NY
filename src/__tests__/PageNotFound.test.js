import React from 'react';
import renderer from 'react-test-renderer';

import PageNotFound from '../components/PageNotFound';

it('correctly renders About component', () => {  
    const PageNotFoundComponent = renderer.create(<PageNotFound />).toJSON();
    expect(PageNotFoundComponent).toMatchSnapshot();
});