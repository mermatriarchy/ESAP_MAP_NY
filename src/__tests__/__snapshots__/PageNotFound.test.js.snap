// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders About component 1`] = `
<h1>
  Sorry! Page not found.
</h1>
`;
